src/lib/storage/create_iterable_from_segment.ipp

```diff
@@ -50,12 +50,12 @@ template <typename T, bool EraseSegmentType>
 auto create_iterable_from_segment(const FixedStringDictionarySegment<T>& segment) {
 #ifdef HYRISE_ERASE_FIXEDSTRINGDICTIONARY
   PerformanceWarning("FixedStringDictionarySegmentIterable erased by compile-time setting");
-  return AnySegmentIterable<T>(DictionarySegmentIterable<T, FixedStringVector>(segment));
+  return AnySegmentIterable<T>(DictionarySegmentIterable<T, FixedStringVector, std::span<const int>>(segment));
```

Comment:
Once "we are done" with the dictionaries, the second template parameter can get removed?

Reply:
If we get rid of vectors inside the segments then yes.